RecoParticleFlow/PFTracking/src/PFDisplacedVertexCandidateFinder.cc

```diff
@@ -191,7 +192,7 @@ void PFDisplacedVertexCandidateFinder::link(const TrackBaseRef& el1,
     dist = -1;
     return;
   }
-  if (pt1 > 2 && pt2 > 2 && std::abs(phi1 - phi2) > 1) {
+  if (pt1 > 2 && pt2 > 2 && ::deltaPhi(phi1, phi2) > 1) {
```

Comment:
IIUC deltaPhi is signed. So, an abs is still needed

Reply:
done in the last push.